examples/benchmark/benchmark.js

```diff
@@ -21,7 +21,13 @@ suite
     uuidv1();
   })
   .add('uuidv1() fill existing array', function () {
-    uuidv1(null, array, 0);
+    try {
+      uuidv1(null, array, 0);
+    } catch (err) {
+      if (err.code !== 'UUID_V1_LIMIT_PER_SEC') {
+        throw err;
+      }
+    }
```

Comment:
Revert?

Why is this needed?  Are we hitting the rate limit with our benchmark?  If so, that's kind of cool... but it also invalidates the benchmark so should probably be `abort()`ed with an appropriate message.

Note: Silently dropping errors in `catch` blocks is a very strong code smell.  At a minimum, there should be a comment explaining the rationale for it.

Reply:
@broofa periodically the limit is exceeded on my machine and an exception is thrown.
This is strange, but the block `try/catch` does not affect the performance %)
If an exception is thrown, the following tests will fail.

In new versions of nodejs (v13.14.0), performance is even higher. Exceptions are thrown more often.

Reply:
> periodically the limit is exceeded on my machine

This is surprising.  It's the first time I've seen this limit hit in the real world.  (I added that rate check years ag to comply with the spec, never really expecting to hit it in practice.)  What hardware are you running on if I may ask?

Reply:
@broofa I have my own computer on which I run tests (nodejs v12.16.3):

CPU: AMD Ryzen 7 1700
RAM: 32 Gb 2999 MHz

Typically, an exception occurs the first few times the benchmark is launched. And then the testing goes fine. Maybe it's a bug. (With `try/catch` I never get a performance close or higher to 10M)

The exception is much more likely to occur on new versions of nodejs (v13). Because the overall performance of the library is increasing.

On nodejs v13 I usually get performance: `uuidv1() fill existing array x 8,777,105 ops/sec ±0.82% (96 runs sampled)`.